File: C & C++ PRACTICE CODES/Pyramid Pattern/code2.c
#include <stdio.h>

void main(){

	int row , count =1 ;
	printf("enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 1 ; i  <= row ; i++){

		for(int a = 0; a < row -i ;a++ ){

			printf(" \t");
		}

		for(int j = 0 ; j < (2*i)-1; j++ ){


			printf("%d\t",count++);
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/7/code2.java
import java.util.Scanner;

class NestedForDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter string : ");
		String str = s.nextLine();
		System.out.println("Enter no. of rows : ");
		int row = s.nextInt();

		for(int i = 0; i < row ; i++){

			for(int j = 0; j < row ; j++){

				System.out.print(str+"	");

			}
			System.out.println("\n");

		}

	}


}
